tark/tark/templates/documentation.html

```diff
@@ -1,11 +1,91 @@
+<head>
+	<style>
+	<!---------------------------------------------------------------------------------->
```

Comment:
Few remarks:

- `<!--` [are not recommended in CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/Comments#notes) and result in compilation errors, use `/* */` instead.
- One simple line is enough (e.g. `/* Add CSS for collapsible */`)